src/common/utils/gpo.h

```diff
@@ -286,47 +287,47 @@ namespace powertoys_gpo {
 
     inline gpo_rule_configured_t getConfiguredSvgPreviewEnabledValue()
     {
-        return getUtilityEnabledValue(POLICY_CONFIGURE_ENABLED_SVG_PREVIEW);
+        return getConfiguredValue(POLICY_CONFIGURE_ENABLED_SVG_PREVIEW);
```

Comment:
Why are we making these utilities no longer respond to the "All utilities" GPO?

Reply:
It was based on how it is working now. We discussed this in the PR discussion. They are handled as sub settings now.

And the new "main" utility GPO is the one for "File Explorer add-ons module". And this new policy is respecting the "all utilities gpo".

If the previewers itself tespect the all utilities gpo they can override the explorer add-ons module gpo. And this can lead to confusing behavior.

Reply:
@jaimecbernardo 
For clarification. The problem is not that individual previewers will override the "All Utilities GPO". The problem is that with "All Utilities GPO" being disabled and "FileExplorer Addons Module GPO" being enabled, the resulting behavior is strange. 

#### Example:
- All Utillities GPO = disabled
- File Explorer Add-ons Module enabled state = enabled
- Individual File Explorer add-ons enabled state = All are not configured.
- Result: The module is enabled and no individual previewer works as they are still disabled by the "All Utilities GPO".

### Possible solution for the problem:
If we need a policy to disable new/unknown previewers like the old behavior was, I suggest the following: Add a new policy like we have for unmanaged PT Run plugins. We can name it "Default enabled state for individual File Explorer Add-ons" and place it in the "File Explorer" category. (Then we are as flexible as possible.)
The behavior would be:
- Not configured: User takes control.
- Enabled: Add-on is enabled.
- Disabled: Add-on is disabled.

This solution fixes the conflict we have when using the "All Utilities GPO" while still providing the possibility to handle newly added Add-ons by default.